sympy/solvers/tests/test_solvers.py

```diff
@@ -2314,3 +2314,16 @@ def test_issue_20902():
     assert solve(f.subs({t: 3 * x + 3}).diff(x) > 0, x) == (S(-4)/3 < x) & (x < S(-2)/3)
     assert solve(f.subs({t: 3 * x + 4}).diff(x) > 0, x) == (S(-5)/3 < x) & (x < S(-1))
     assert solve(f.subs({t: 3 * x + 2}).diff(x) > 0, x) == (S(-1) < x) & (x < S(-1)/3)
+
+def test_issue_4886():
+    ra = Symbol('ra', positive=True)
```

Comment:
There should be two blank lines between `test_` functions. Can you also fix this for the functions immediately above?